### PR TITLE
Implement discord notifications

### LINKED ISSUE #17: discord notifications

### PR DESCRIPTION
**Pull request by made-by-traxam**
Closes #17

### CODE REVIEW

File: packages/web-backend/src/modules/cfg.ts [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
 
+/**
+ * Configuration data for loader, launcher and mod release notifications to
+ * Discord webhooks.
+ */
+export interface DiscordNotificationsCfg {
+  /**
+   * Minimum time distance between role pings in milliseconds.
+   */
+  pingCooldown: number;
```
Comment (Felix-Puetz) [inline packages/web-backend/src/modules/cfg.ts:11]: if its a webhook, why do we need a ping cooldown?

**Review by Felix-Puetz — CHANGES_REQUESTED** [review]
Are you sureyou want that many "utils"  method that have hard-coded uris?

File: packages/web-backend/src/modules/cfg.ts (hunk 1 above)
Comment (made-by-traxam) [reply packages/web-backend/src/modules/cfg.ts:11]: The cooldown is a requirement from @TeKGameR950. The basic idea is that if a lot of notifications are sent out, not every single one of them will have an @role mention with it.

**Comment by made-by-traxam** [comment]
> Are you sure you want this many "utils" method that have hard-coded URIs?

Is there a better place for stuff like that? I didn't want to put these constant strings to random places in the code.

**Comment by Felix-Puetz** [comment]
We'lol leave it for now 😂

**Review by Felix-Puetz — APPROVED** [review]

**Pull request merged**